bool active
float32 rads_per_second